demos/pet-shop/src/app.js

```diff
@@ -1,113 +1,117 @@
-
-        petsRow.append(petTemplate.html());
+        petTemplate.find('.panel-title').text(data[i].name)
+        petTemplate.find('img').attr('src', data[i].picture)
+        petTemplate.find('.pet-breed').text(data[i].breed)
+        petTemplate.find('.pet-age').text(data[i].age)
+        petTemplate.find('.pet-location').text(data[i].location)
+        petTemplate.find('.btn-adopt').attr('data-id', data[i].id)
+
+        petsRow.append(petTemplate.html())
       }
-    });
+    })
 
-    return await App.initWeb3();
+    return await App.initWeb3()
   },
 
-  initWeb3: async function() {
+  initWeb3: async function () {
     // Modern dapp browsers...
-    let web3Provider = null;
+    let web3Provider = null
     if (window.ethereum) {
-      web3Provider = window.ethereum;
+      web3Provider = window.ethereum
       try {
         // Request account access
-        await window.ethereum.enable();
+        await window.ethereum.enable()
       } catch (error) {
         // User denied account access...
-        console.error("User denied account access");
+        console.error('User denied account access')
       }
     }
     // Legacy dapp browsers...
     else if (window.web3) {
-      web3Provider = window.web3.currentProvider;
+      web3Provider = window.web3.currentProvider
     }
     // If no injected web3 instance is detected, fall back to Ganache
     else {
-      web3Provider = new Web3.providers.HttpProvider("http://localhost:7545");
+      web3Provider = new Web3.providers.HttpProvider('http://localhost:7545')
     }
 
-    let provider = ArbProvider("http://localhost:1235", web3Provider);
-    App.web3 = new Web3(provider); // eslint-disable-line require-atomic-updates
+    let provider = ArbProvider(
+      'http://localhost:1235',
+      web3Provider,
+      'http://localhost:1237'
+    )
+    App.web3 = new Web3(provider) // eslint-disable-line require-atomic-updates
 
-    return App.initContract();
+    return App.initContract()
   },
 
-  initContract: function() {
-    let adoption = require("../build/contracts/Adoption.json");
+  initContract: function () {
+    let adoption = require('../build/contracts/Adoption.json')
     App.contracts.Adoption = new App.web3.eth.Contract(
       adoption.abi,
-      adoption.networks["123456789"].address
-    );
+      adoption.networks['123456789'].address
+    )
 
     // Use our contract to retrieve and mark the adopted pets
-    App.markAdopted();
+    App.markAdopted()
 
-    return App.bindEvents();
+    return App.bindEvents()
   },
 
-  bindEvents: function() {
-    $(document).on("click", ".btn-adopt", App.handleAdopt);
+  bindEvents: function () {
+    $(document).on('click', '.btn-adopt', App.handleAdopt)
   },
 
-  markAdopted: async function() {
+  markAdopted: async function () {
     try {
-      let adopters = await App.contracts.Adoption.methods.getAdopters().call();
+      let adopters = await App.contracts.Adoption.methods.getAdopters().call()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['adopters' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)